content/shmem_reductions.tex

```diff
@@ -200,8 +200,8 @@
     and \source{} arrays, and the same \VAR{pWrk} and \VAR{pSync} work arrays, must
     be passed to all \acp{PE} in the \activeset.
     
-    Before any \ac{PE} calls a reduction routine, you must ensure that the
-    following conditions exist (synchronization via a \OPR{barrier} or some other
+    Before any \ac{PE} calls a reduction routine, the
+    following conditions must be met (synchronization via a \OPR{barrier} or some other
```

Comment:
Mismatched consistency with other comment (Ref.1) of the text in parentheses. I don't know what the OPR command is used for or whether this phrasing is used in other collectives (e.g., collect, all2all).